lib/models/grid_items_list.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class GridItemModel {
+  String name;
+  String image;
+  final price;
+  GridItemModel({
+    required this.name,
+    required this.image,
+    required this.price,
+  });
+}
```

Comment:
Models attributes are recommended to be final and constructors to be const

Like this:

```
class GridItemModel {
  final String name;
  final String image;
  final price;
  const GridItemModel({
    required this.name,
    required this.image,
    required this.price,
  });
}
```